packages/acceptance-tests/pkg-tests-specs/sources/commands/version.test.js

```diff
@@ -0,0 +1,21 @@
+describe(`Commands`, () => {
+  describe(`version`, () => {
+
+    test(
+      `it should print the version from the package.json when given --version`,
+       makeTemporaryEnv({}, async ({path, run, source}) => {
+        const {stdout} = await run(`--version`);
+        expect(stdout).toMatchSnapshot();
```

Comment:
That will require use to update the snapshots each time we make a release; I suggest we instead check that it matches a basic regexp (like `/^v[0-9]+\.[0-9]+\.[0-9]+\n$/`).